code/game/turfs/turf.dm

```diff
@@ -185,7 +185,7 @@ GLOBAL_LIST_EMPTY(station_turfs)
  * * source_atom - If this is not null, will check whether any contents on the turf can block this atom specifically. Also ignores itself on the turf.
  * * ignore_atoms - Check will ignore any atoms in this list. Useful to prevent an atom from blocking itself on the turf.
  */
-/turf/proc/is_blocked_turf(exclude_mobs = FALSE, source_atom = null, list/ignore_atoms)
+/turf/proc/is_blocked_turf(exclude_mobs = FALSE, ignore_climbable = FALSE, source_atom = null, list/ignore_atoms)
```

Comment:
Please don't expand on this proc further. It will be a mess to untangle, and it's answering to too many situations already.
Counter-intuitively as it may sound, if your really need it do another different custom check pro. That will be easier to refactor.